#: __init__.py:96
msgid "NextGIS geoinformation system"
msgstr "Sistema de geoinformação NextGIS"

#: __init__.py:218
msgid "Linux kernel"
msgstr "Kernel Linux"

#: __init__.py:221
msgid "OS distribution"
msgstr "Distribuição do sistema operativo"

#: __init__.py:233
msgid "CPU"
msgstr "CPU"

#: __init__.py:237
msgid "RAM"
msgstr "RAM"

#: exception.py:90
msgid "Validation error"
msgstr "Erro de validação"

#: exception.py:95
msgid "Forbidden"
msgstr ""

#: exception.py:100
msgid "Insufficient permissions"
msgstr "Permissões insuficientes"

#: exception.py:105
msgid "Operational error"
msgstr "Erro operacional"

#: storage.py:222
msgid "Estimation is already running"
msgstr ""

#: test/test_exception.py:75
#, python-format
msgid "The answer is %d"
msgstr "A resposta é %d"

